ubg/src/pages/Room.js

```diff
@@ -224,9 +225,10 @@ export default function Room() {
                     user={user}
                     usersData={usersData}
                     room={room}
-                    mafiaKill={roomData.mafiaKill}
-                    doctorSave={roomData.doctorSave}
-                    detectiveCheck={roomData.detectiveCheck}
+                    mafiaKill={usersCollection.mafiaKill}
```

Comment:
Isn't usersCollection the ref to to collection?

Reply:
I wasn't sure about syntax so I followed the comment above. I can change it back.

Reply:
I think it is in roomData though